### PR TITLE
Add new field and changed its data structure

### PR DESCRIPTION
**Pull request by cekk**
…and changed its data structure

### CODE REVIEW

File: src/redturtle/voltoplugin/editablefooter/restapi/configure.zcml [hunk 1]
```diff
@@ -23,14 +23,14 @@
   <!-- endpoint to get footer data -->
   <plone:service
       method="GET"
-      factory=".get.FooterColumns"
+      factory=".get.EditableFooterGet"
       for="Products.CMFCore.interfaces.ISiteRoot"
       permission="zope2.View"
-      name="@footer-columns"
+      name="@editable-footer-data"
```
Comment (mamico) [inline src/redturtle/voltoplugin/editablefooter/restapi/configure.zcml:29]: why not leave a deprecated view with the old name for a while?

File: src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    >
+
+  <genericsetup:upgradeStep
+      title="Add new field in controlpanel"
+      description=""
+      profile="redturtle.voltoplugin.editablefooter:default"
+      source="1000"
+      destination="1001"
+      handler=".upgrades.update_registry"
```
Comment (mamico) [inline src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml:12]: You are updating the registry, but I don't see any files added/changed, did you forget in the commit?

Comment (cekk) [reply src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml:12]: i added a new field in interfaces.py

Comment (cekk) [reply src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml:12]: i don't know (never done) if there is a way to add it manually

**Review by luca-bellenghi — APPROVED** [review]

File: src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml (hunk 2 above)
Comment (luca-bellenghi) [reply src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml:12]: afaik you need to reload registry

**Pull request merged**